ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml

```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-aws-ci-cdv2-test-252nodes-scale-ovnic
+- as: perfscale-aws-ci-control-plane-test-252nodes-scale-ovnic
```

Comment:
We lose history when the tests are renamed. For the sake of traceability and with these ovnic tests being temporary, I request that we don't change any of the job names.

Reply:
@svetsa-rh @afcollins Prow jobs can only be 61 characters long and `ocp-qe-perfscale-aws-ci-control-plane-test-252nodes-scale-ovnic` does not make the cut,

> with these ovnic tests being temporary, I request that we don't change any of the job names.
Ack, will keep the job name relevant to the workload we are running, and will find a way to keep it <=61 characters long.

Reply:
I want to push back on running node-density and CNI.